gamestonk_terminal/portfolio/port_controller.py

```diff
@@ -188,6 +203,42 @@ def call_dghold(self, _):
             print(e)
             print("")
 
+    def call_dgtopnews(self, _):
```

Comment:
Here you should add a docstring like:
"""" Process dgtopnews command """

Reply:
Added